### PR TITLE
Server - create entry point for app (server.js)

### PR DESCRIPTION
**Pull request by OfirArbel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Dependency directories
+node_modules
```
Comment (nirgn975) [inline .gitignore]: Create `.gitignore` file with https://www.gitignore.io/ website.

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var mongoose = require('mongoose');
+var app = express();
+
+
+module.export = mongoose.connect('mongodb://localhost/users');
```
Comment (nirgn975) [inline server/server.js]: Get the `mongodb://localhost/users` from an environment variable.

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+app.listen(3000);
```
Comment (nirgn975) [inline server/server.js]: Get the port number from an environment variable.

File: server/server.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+console.log('on port 3000');
```
Comment (nirgn975) [inline server/server.js]: Use the `3000` from the environment variable you got earlier.

File: .gitignore (hunk 1 above)
Comment (OfirArbel) [reply .gitignore]: Tnk GEVER

**Comment by nirgn975** [comment]
@OfirArbel Add the new `.env` file anyone should create now to the `README.md` file. So if one `clone` this repo he will know what to create and where so the app will work.

**Pull request merged**